synth.py

```diff
@@ -21,34 +21,42 @@
 
 logging.basicConfig(level=logging.DEBUG)
 
-gapic = gcp.GAPICGenerator()
-library = gapic.node_library(
-    'iot', 'v1',
-    config_path="/google/cloud/iot/"
-                f"artman_cloudiot.yaml")
+gapic = gcp.GAPICMicrogenerator()
+versions = ['v1']
+
+for version in versions:
+    library = gapic.typescript_library(
+        'iot', 
+        generator_args={
+            "grpc-service-config": f"google/cloud/iot/{version}/cloudiot_grpc_service_config.json",
+            "package-name": f"@google-cloud/iot"
+            },
+        proto_path=f'/google/cloud/iot/{version}',
+        extra_proto_files=['google/cloud/common_resources.proto'],
+        version=version)
 
 # skip index, protos, package.json, and README.md
 s.copy(
     library,
-    excludes=['package.json', 'README.md', 'src/index.js'],
+    excludes=['package.json', 'README.md', 'src/index.ts'],
 )
 
 # Streaming is broken and missing grpc handling
 # https://github.com/googleapis/gapic-generator/issues/2152
-s.replace(
-    'src/v1/device_manager_client.js',
-    f'(listDeviceRegistriesStream\(.*\n\s+options = .*\n)(\n\s+return)',
-    '''\g<1>options.otherArgs = options.otherArgs || {};
-    options.otherArgs.headers = options.otherArgs.headers || {};
-    options.otherArgs.headers[
-      'x-goog-request-params'
-    ] = gax.routingHeader.fromParams({
-      parent: request.parent,
-    });\n\g<2>''')
+# s.replace(
```

Comment:
If we don't need this anymore, let's remove it.  Hard to say what it was originally doing ....

Reply:
The issue it refers to is closed long time ago. We should just remove this comment.

Reply:
understood, removed it, thanks